src/Microsoft.Azure.WebJobs.Host/Indexers/FunctionIndexer.cs

```diff
@@ -329,22 +368,10 @@ private bool HasJobAttribute(CustomAttributeData attributeData)
                     parameters.Add(nonTriggerBindings[name].ToParameterDescriptor());
                 }
             }
+                        
+            descr.Parameters = parameters;
```

Comment:
Can we keep all of the FunctionDescriptor initialization within one method? Depending on which method is used parameters could be incorrect.